samples/core/Modeling/EntityProperties/DataAnnotations/Required.cs

```diff
@@ -8,13 +8,11 @@ internal class MyContext : DbContext
         public DbSet<Blog> Blogs { get; set; }
     }
 
-    #region Required
```

Comment:
I didn't mean to remove it here. I meant it should not be in the new sample. It is required here (no pun intended) to define which part of the code to show in the docs.